e2e/testdata/fn-render/invalid-subpkg/deployment_httpbin.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
```

Comment:
nit file name uses undercore instead of dash

Reply:
Removed the file in this case. But found the root cause of this naming, `kyaml` library uses `_` delimiter for generated resource.